packages/oid4vci-demo-frontend/src/configs/knb.json

```diff
@@ -2,7 +2,8 @@
   "general": {
     "credentialName": "Sphereon Guest",
     "issueCredentialType": "GuestCredential",
-    "baseUrl": "https://ssi.sphereon.com/demo/issuer",
+    "agentVpBaseUrl": "https://knb.demo.sphereon.com/agent",
```

Comment:
does knb have it's own issuer and verifier?

Reply:
Knb only has a landingpage so actually I can remove them